File: .php/Program/Aksesoris.php
<?php
require_once 'PetShop.php';

class Aksesoris extends PetShop {
    private $jenis = '';
    private $bahan = '';
    private $warna = '';

    public function __construct($ID = 0, $nama = '', $stok = 0, $harga = 0, $foto = '', $jenis = '', $bahan = '', $warna = '') {
        parent::__construct($ID, $nama, $stok, $harga, $foto);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function setJenis ($jenis) { $this->jenis = $jenis; }
    public function getJenis () { return $this->jenis; }

    public function setBahan ($bahan) { $this->bahan = $bahan; }
    public function getBahan () { return $this->bahan; }

    public function setWarna ($warna) { $this->warna = $warna; }
    public function getWarna () { return $this->warna; }
}
?>

File: .cpp/Program/Aksesoris.cpp
#include "PetShop.cpp"

class Aksesoris: public PetShop {
private:
    string jenis;
    string bahan;
    string warna;

public:
    Aksesoris () {
        jenis = "";
        bahan = "";
        warna = "";
    };

    Aksesoris (string jenis, string bahan, string warna) {
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    void setJenis (const string& jenis) { this->jenis = jenis; }
    string getJenis () { return this->jenis; }
    
    void setBahan (const string& bahan) { this->bahan = bahan; }
    string getBahan () { return this->bahan; }
    
    void setWarna (const string& warna) { this->warna = warna; }
    string getWarna () { return this->warna; }

    ~Aksesoris() { }
};